src/main/java/Bobby.java

```diff
@@ -319,28 +320,28 @@ private static void handle_event() throws Exception {
     private static void check_action(ActionType action) throws Exception {
         switch (action) {
             case bye:
-                handle_bye();
+                handleBye();
                 break;
             case list:
-                handle_list();
+                handleList();
                 break;
             case delete:
-                handle_delete();
+                handleDelete();
                 break;
             case mark:
-                handle_mark();
+                handleMark();
                 return;
             case unmark:
-                handle_unmark();
+                handleUnmark();
                 return;
             case todo:
-                handle_todo();
+                handleTodo();
                 break;
             case deadline:
-                handle_deadline();
+                handleDeadline();
                 break;
```

Comment:
Should the "case" statements be indented at the same level as the switch statement to follow the coding convetion?